<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1063</width>
    <height>726</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="nextButton">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>520</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Suivant &gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="previousButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>520</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt; Précédent</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>791</width>
      <height>461</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="kanjiLabel">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>70</y>
       <width>211</width>
       <height>121</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>48</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Kanji</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="idLabel">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>10</y>
       <width>141</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>ID</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lvlLabel">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>180</y>
       <width>141</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>LVL</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="readingLabel">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>350</y>
       <width>141</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>Reading</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="translationLabel">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>240</y>
       <width>141</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>Translation</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>620</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Manage Data</string>
    </property>
   </widget>
   <widget class="QPushButton" name="answerButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>520</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Reveal Answers</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>500</y>
      <width>791</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1063</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
